### PR TITLE
MM-41143: Pass a custom logger to morph

### PR DESCRIPTION
**Pull request by agnivade**
We create a custom io.Writer to pass to the logger
instance. For now, we keep everything to debug
as all errors are surfaced back to the server via
the API.

```release-note
NONE
```

### CODE REVIEW

File: store/sqlstore/utils.go [hunk 1]
```diff
@@ -147,3 +147,18 @@ func constructArrayArgs(ids []string) (string, []interface{}) {
...
+
+// morphWriter is a target to pass to the logger instance of morph.
+// For now, everything is just logged at a debug level. If we need to log
+// errors/warnings from the library also, that needs to be seen later.
+type morphWriter struct {
+}
+
+func newMorphWriter() *morphWriter {
+	return &morphWriter{}
+}
```
Comment (isacikgoz) [inline store/sqlstore/utils.go:159]: Do we need this function? It seems to be redundant.

File: store/sqlstore/utils.go [hunk 2]
```diff
@@ -147,3 +147,18 @@ func constructArrayArgs(ids []string) (string, []interface{}) {
...
+
+func (l *morphWriter) Write(in []byte) (int, error) {
```
Comment (isacikgoz) [inline store/sqlstore/utils.go:161]: I'd suggest removing the escape sequences for colors, right now the log line looks like: 

`"morph.go:174: \u001b[36m==  upgrade_users_v6.3: migrated (0.0025s)  ========================================================\u001b[0m\n","caller":"sqlstore/utils.go:158"`

Is it something doable?

**Review by isacikgoz — COMMENTED** [review]
I have a couple cosmetic related question. Otherwise looks good 🚀

File: store/sqlstore/utils.go (hunk 2 above)
Comment (agnivade) [reply store/sqlstore/utils.go:161]: https://github.com/go-morph/morph/pull/37 :)

File: store/sqlstore/utils.go (hunk 1 above)
Comment (agnivade) [reply store/sqlstore/utils.go:159]: Hmm okay, will remove.

File: store/sqlstore/utils.go (hunk 2 above)
Comment (isacikgoz) [reply store/sqlstore/utils.go:161]: Ehe, didn't see that. Well, I think we can handle this on the morph side. Eg. if another logger passed to replace default one we can disable coloring. In general i think coloring helps.

Comment (agnivade) [reply store/sqlstore/utils.go:161]: Yes exactly.

**Review by isacikgoz — APPROVED** [review]
Awesome 💯

**Pull request merged**